node/packages/aws-lambda-sdk/instrument/lib/send-telemetry.js

```diff
@@ -33,10 +33,12 @@ module.exports = limit(10, async (name, body) => {
         },
         (response) => {
           if (response.statusCode !== 200) {
-            process.stderr.write(
-              'Serverless SDK Warning: Cannot propagate telemetry, ' +
-                `server responded with "${response.statusCode}" status code\n`
-            );
+            console.warn({
+              source: 'serverlessSdk',
+              message:
+                'Cannot propagate telemetry, ' +
+                `server responded with "${response.statusCode}" status code\n`,
+            });
```

Comment:
These warnings might be nice to search for in Cloudwatch so maybe it also make sense to put these variables in the string into the warning object? Maybe this one under `statusCode`?

Reply:
That's a good hint. Although I'm not sure how useful it would be to search here on specific status codes.
What would definitely be nice is to narrow the search to all warnings of the same type, and for that, I've added `code` property (similar concept as there's with error).

Nonetheless I've also added `httpStatusCode` in this case.

Reply:
Oh yeah I like what you have added. That would make it easy to find these if they ever came up. Thanks man 👍